services/nrfradio.md

```diff
@@ -0,0 +1,36 @@
+# NRF Radio
+
+    identifier: 0x1ac986cf
+
+## Registers
+
+    rw enabled: bool @ intensity
+
+Turns on/off the radio antenna.
+
+    rw group: u8 @ 0x80
+
+Group used to filter packets
+
+    rw transmission_power = 6: u8 { min_value=1, max_value=7 } @ 0x81
+
+Antenna power to increase or decrease range.
+
+## Commands
+
+    command send @ 0x80 {
+        data: bytes { max_bytes: 18 }
+    }
+
+Sends a payload of bytes as a radio message
+
+## Events
+
+    event packet @ 0x80 {
+        time: u32 ms
+        rssi: i16 db
+        device_serial_number: u32
```

Comment:
I think the device serial is m:b-specific; in fact there seem to be some addresses in the packets themselves, maybe these are MAC addresses of BLE? @jamesadevine ?

Reply:
AFAIK, micro:bit does not include the mac address of the hardware to make PII data in packets hard to trace back to a source. The group id (address) provides a group abstraction for user applications. If some sort of identifier is required it is consciously added by the user. The radio hardware can filter based on the given group id (address).

See: https://github.com/lancaster-university/microbit-dal/blob/master/inc/drivers/MicroBitRadio.h#L80

Reply:
I'm renaming this to "bit:radio". this is specifically to communicate with other microbits.